README.md

```diff
@@ -1,19 +1,148 @@
+
+In addition, one missing option is added to the existing `currencyDisplay` setting: "narrowSymbol", which uses the CLDR narrow-format symbol:
+
+```javascript
+(100).toLocaleString("en-CA", {
+    style: "currency",
+    currency: "USD",
+    currencyDisplay: "narrowSymbol"
+});
+// ==> "$100" (rather than "US$100")
+```
+
+## II. Measure Units
+
+Units of measurement can be formatted as follows:
+
+```javascript
+(9.81).toLocaleString("en-US", {
+    style: "unit",
+    unit: "acceleration-meter-per-second-squared",
+    unitDisplay: "short"
+});
+// ==> "9.81 m/s²"
+```
+
+The syntax was discussed in #3.
+
+- `style` receives the string value "unit"
+- `unit` receives a string measure unit identifier, defined in [UTS #35](http://unicode.org/reports/tr35/tr35-general.html#Unit_Elements).  See also the [full list of unit identifiers](https://unicode.org/repos/cldr/tags/latest/common/validity/unit.xml).
+    - Note: this list is not "arbitrary generative"; this is the shape of the data we have.
+- `unitDisplay`, named after the corresponding setting for currencies, `currencyDisplay`, takes either "narrow", "short", or "long".
+
+## III. Scientific and Compact Notation
+
+Scientific and compact notation are represented by the new option `notation` and can be formatted as follows:
+
+```javascript
+(987654321).toLocaleString("en-US", {
+    notation: "scientific"
+});
+// ==> 9.877E8
+
+(987654321).toLocaleString("en-US", {
+    notation: "engineering"
+});
+// ==> 987.7E6
+
+(987654321).toLocaleString("en-US", {
+    notation: "compact",
+    compactDisplay: "long"
+});
+// ==> 987.7 million
+```
+
+The syntax was discussed in #5.
+
+- `notation` takes either "scientific", "engineering", "compact", or "plain"
+- `compactDisplay`, used only when `notation` is "compact", takes either "short" or "long"
+
+Rounding-related settings (min/max integer/fraction digits) are applied after the number is scaled according to the chosen notation.
+
+Notation styles are allowed to be combined with other options:
+
+```javascript
+(299792458).toLocaleString("en-US", {
+    notation: "scientific",
+    minimumFractionDigits: 2,
+    maximumFractionDigits: 2,
+    style: "unit",
+    unit: "speed-meter-per-second"
+});
+// ==> 3.00E8 m/s
+```
+
+## IV. Sign Display
+
+The sign can be displayed on positive numbers:
+
+```javascript
+(55).toLocaleString("en-US", {
+    signDisplay: "always"
+});
+// ==> +55
+```
+
+Currency accounting sign display is also supported via a new option. In many
+locales, accounting format means to wrap the number with parentheses instead of
+appending a minus sign.  For example:
+
+```javascript
+(-55).toLocaleString("bn", {
+    style: "currency",
+    currency: "EUR",
+    currencySignDisplay: "accounting"
+});
+// ==> (১০০.০০€)
+```
+
+The syntax was discussed in #6.
+
+- `signDisplay`: "auto" (default), "always", "never", "except-zero"
+- `currencySignDisplay`: "standard" (default), "accounting"
```

Comment:
Explaining currencySignDisplay here would be good (copy-pasting the explanation from the specification would work; many people who read the README won't read the spec).

Reply:
Added a pointer to the example above.